src/main/java/org/classes/DBOperations.java

```diff
@@ -0,0 +1,64 @@
+package src.main.java.org.classes;
+
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.bson.Document;
+import org.bson.conversions.Bson;
+public class DBOperations {
+    private com.mongodb.client.MongoClient mongoClient;
+    private MongoDatabase database;
+    private MongoCollection<Document> userStoryCollection;
+
+    public DBOperations(String connectionString, String dbName, String collectionName) {
+        mongoClient = MongoClients.create(connectionString);
+        database = mongoClient.getDatabase(dbName);
+        userStoryCollection = database.getCollection(collectionName);
+    }
+
+    // Adding/Creating a US in the Database
+    public void addUserStory(UserStory US) {
+        Document userStoryDocument = new Document();
+        userStoryDocument.put("uid", US.getId());
+        userStoryDocument.put("title", US.getTitle());
+        userStoryDocument.put("description", US.getDescription());
+        userStoryDocument.put("acceptanceCriteria", US.getAcceptanceCriteria());
+        userStoryDocument.put("businessValue", US.getBusinessValue());
+        userStoryDocument.put("developerValue", US.getDeveloperValue());
+
+        userStoryCollection.insertOne(userStoryDocument);
+    }
+
+    // To Fetch a US from DB by ID
+    public Document getUserStoryById(int id) {
+        Bson filter = new Document("_id", id);
+        return userStoryCollection.find(filter).first();
```

Comment:
add a check to see if there are multiple versions of the same user story.. ( for future don't have to do it right away )